### ISSUE TITLE
KeyError 

### ISSUE THREAD

**Issue by tobi-nmx** [LeoVerto/docker-network-graph#11]
When executing 
`docker run --rm -v /var/run/docker.sock:/var/run/docker.sock leoverto/docker-network-graph`
I get this:
```
Traceback (most recent call last):
  File "docker-net-graph.py", line 211, in <module>
    generate_graph(args.verbose, args.out)
  File "docker-net-graph.py", line 190, in generate_graph
    draw_link(g, networks, link)
  File "docker-net-graph.py", line 166, in draw_link
    color=networks[link.network_name].color
KeyError: 'mailcowdockerized_mailcow-network'
```

Here's the definition of the offending network (from https://github.com/mailcow/mailcow-dockerized/):
```
networks:
  mailcow-network:
    driver: bridge
    driver_opts:
      com.docker.network.bridge.name: br-mailcow
    enable_ipv6: true
    ipam:
      driver: default
      config:
        - subnet: ${IPV4_NETWORK:-172.22.1}.0/24
        - subnet: ${IPV6_NETWORK:-fd4d:6169:6c63:6f77::/64}
```

Thanks for looking into this!

**Comment by LeoVerto** [comment]
This issue should be fixed in 8c5cc44 but please be aware that I haven't found a good way to handle dual-stack networks yet.
The current assumption is that each network only has one subnet, so any further ones won't be displayed.
I'll have to look into good ways for displaying the IPv6 addresses.

**Comment by tobi-nmx** [comment]
Thanks for the fast fix! The assumption is probably true for most networks. Works for me now!
